qa/tasks/qemu.py

```diff
@@ -104,6 +112,19 @@ def create_clones(ctx, config, managers):
                 rbd.clone_image(ctx=ctx, config=create_config)
                 )
 
+def create_encrypted_devices(ctx, config, managers):
+    for client, client_config in config.items():
+        disks = client_config['disks']
+        for disk in disks:
+            if disk['encryption_format'] == 'none':
```

Comment:
Nit: noticed that the 3 parent images were being NBD mapped as well, so perhaps this should also skip mapping the devices if `'device_letter' not in disk`